src/test/java/uk/gov/hmcts/probate/validator/CaveatorEmailAddressValidationRuleTest.java

```diff
@@ -0,0 +1,169 @@
+package uk.gov.hmcts.probate.validator;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.probate.exception.BusinessValidationException;
+import uk.gov.hmcts.probate.model.ApplicationType;
+import uk.gov.hmcts.probate.model.ccd.caveat.request.CaveatData;
+import uk.gov.hmcts.probate.model.ccd.caveat.request.CaveatDetails;
+import uk.gov.hmcts.probate.service.BusinessValidationMessageRetriever;
+
+public class CaveatorEmailAddressValidationRuleTest {
+
+    @InjectMocks
+    private CaveatorEmailAddressValidationRule caveatorEmailAddressValidationRule;
+
+    @Mock
+    private BusinessValidationMessageRetriever businessValidationMessageRetriever;
+
+    private CaveatData CaveatDataEmptyEmail;
+    private CaveatData CaveatDataEmptyEmailNull;
+    private CaveatData CaveatDataEmailInvalid1;
+    private CaveatData CaveatDataEmailInvalid2;
+    private CaveatData CaveatDataEmailInvalid3;
+    private CaveatData CaveatDataEmailInvalid4;
+    private CaveatData CaveatDataEmailValid1;
+    private CaveatData CaveatDataEmailValid2;
+
+    private static final String[] LAST_MODIFIED = {"2020", "1", "1", "0", "0", "0", "0"};
+    private static final Long CASE_ID = 12345678987654321L;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+
+        CaveatDataEmptyEmail = CaveatData.builder()
+                .applicationType(ApplicationType.SOLICITOR)
+                .caveatorEmailAddress("")
+                .registryLocation("Bristol")
+                .build();
+
+        CaveatDataEmptyEmailNull = CaveatData.builder()
+                .applicationType(ApplicationType.SOLICITOR)
+                .caveatorEmailAddress(null)
+                .registryLocation("Bristol")
+                .build();
+
+        CaveatDataEmailInvalid1 = CaveatData.builder()
+                .applicationType(ApplicationType.SOLICITOR)
+                .caveatorEmailAddress("example.@test.com")
```

Comment:
we need to get into the habit of using
XXXX@probate-test.com
as a template for ALL email addresses
even in unit tests

Reply:
Done.